test/test_autograd.py

```diff
@@ -2797,6 +2797,48 @@ def run_test(upper, dims):
         for upper, dims in product([True, False], [(3, 3), (5, 3, 3), (4, 3, 2, 2)]):
             run_test(upper, dims)
 
+    @skipIfNoLapack
+    def test_symeig_degenerate(self):
```

Comment:
Could you please move this test to `test/test_linalg.py`? All linear algebra related functionality belongs to that file.

Reply:
And, please, add support for the batched case.

Reply:
Did it on the last commit.